src/main/docker/vpn/client/Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM ubuntu:14.04
+FROM alpine
```

Comment:
It is not clear to me whether this is a static image or a pointer to latest alpine.
It would be wise to use a specific version where the script is known to work on.